package.json

```diff
@@ -29,16 +29,20 @@
     "@nestjs/mongoose": "^9.2.0",
     "@nestjs/platform-express": "^9.0.0",
     "@nestjs/swagger": "^6.0.5",
+    "@nestjs/typeorm": "^9.0.1",
     "@types/bcrypt": "^5.0.0",
     "bcrypt": "^5.0.1",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.13.2",
     "mongoose": "^6.5.2",
```

Comment:
since we arent using mongodb for now, please remove all packages that have to do with mongo